test/transaction.ts

```diff
@@ -18,12 +18,21 @@ import * as pfy from '@google-cloud/promisify';
 import * as arrify from 'arrify';
 import * as assert from 'assert';
 import * as proxyquire from 'proxyquire';
+import { google } from '../proto/datastore';
+import { RequestConfig } from '../src/request';
+import { Entity } from '../src/entity';
+import { Transaction } from '../src';
+
+//tslint:disable-next-line no-any
+type Any = any;
+type Fn = (...args: Any) => void;
```

Comment:
This exists, it's just called `Function` 😛

Reply:
I was originally using `Function`, but I was like "Maybe someone will want to see what this means" 🤦‍♂️

Fixed!